src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Services/PreviewTokenService.cs

```diff
@@ -85,8 +85,7 @@ private async Task<bool> HasActivePreviewToken(
             .Where(pt => pt.Id == previewTokenId)
             .Where(pt => pt.DataSetVersionId == dataSetVersionId)
             .Where(pt => pt.DataSetVersion.Status == DataSetVersionStatus.Draft)
-            .Where(pt => (pt.Activates == null || pt.Activates <= timestamp) 
-                         && pt.Expiry > timestamp)
+            .Where(pt => pt.Activates <= timestamp && pt.Expires > timestamp)
```

Comment:
Highlighting a change to `Activates` which has crept into this PR. Not sure if this is intentional or not.